--- audiofile-0.3.1/examples/Makefile.am.wiget	2011-10-02 10:24:42.389369020 +0200
+++ audiofile-0.3.1/examples/Makefile.am	2011-10-02 10:26:14.975931303 +0200
@@ -10,7 +10,7 @@ EXTRA_PROGRAMS = alsaplay irixread irixt
 LIBAUDIOFILE = $(top_builddir)/libaudiofile/libaudiofile.la
 
 alsaplay_SOURCES = alsaplay.cpp
-alsaplay_LDADD = $(LIBAUDIOFILE) -lasound
+alsaplay_LDADD = $(LIBAUDIOFILE) -lasound -lm
 
 irixread_SOURCES = irixread.c sgi.c sgi.h
 irixread_LDADD = $(LIBAUDIOFILE) -laudio
@@ -23,6 +23,8 @@ linuxtest_SOURCES = linuxtest.c
 osxplay_SOURCES = osxplay.c
 osxplay_LDFLAGS = -framework AudioUnit -framework CoreServices
 
+power_LDADD = $(LIBAUDIOFILE) -lm
+
 LDADD = $(LIBAUDIOFILE)
 
 DEPENDENCIES = $(LIBAUDIOFILE)
